plugins/afr/validate.go

```diff
@@ -9,12 +9,39 @@ import (
 
 var names = [...]string{"replicate", "afr"}
 
+var shdEnabledKey = [...]string{"afr.self-heal-daemon", "replicate.self-heal-daemon"}
+
+// IsVolReplicate return true if volume is of type replicate, disperse, distreplicate or distdisperse
```

Comment:
For now, duplicate (and unexport) this code in each package where it's used. In future we'll merge afr and selfheal packages.